Docs: Document 'write.parquet.row-group-check-min-record-count' and '… (#6905)
apache/iceberg: docs/configuration.md

| write.parquet.compression-level    | null               | Parquet compression level                          |
| write.parquet.bloom-filter-enabled.column.col1          | (not set) | Enables writing a bloom filter for the column: col1|
| write.parquet.bloom-filter-max-bytes | 1048576 (1 MB)   | The maximum number of bytes for a bloom filter bitset |

> | write.parquet.row-group-check-min-record-count | 100    | The minimum count of rows of checks of the page size and row group size |

Reviewer: ```suggestion
| write.parquet.row-group-check-min-record-count | 100    | The minimum record count for page size check and row group size check|
```

Author: Fixed.

> | write.parquet.row-group-check-max-record-count | 10000  | The maximum count of rows of checks of the page size and row group size |

Reviewer: ```suggestion
| write.parquet.row-group-check-max-record-count | 10000  | The maximum record count for page size check and row group size check|
```

Author: Fixed.
